src/solver/conv_winoRxS_f2x3.cpp

```diff
@@ -44,8 +44,6 @@ MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_AMD_WINOGRAD_RXS_F2X3)
 MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_AMD_WINOGRAD_RXS_F2X3_PERF_VALS)
 MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_AMD_WINOGRAD_RXS_F2X3_G1)
 
-#define WORKAROUND_ISSUE_1093 1
```

Comment:
@Slimakanzer If we are removing this WORKAROUND (which is great, BTW), could we please also update:
> test/CMakeLists.txt

to reflect the changes? e.g. delete line 201-205:
>    if(WORKAROUND_ISSUE_1093)
>           list(APPEND SKIP_TESTS test_find_db test_main test_immed_conv2d)
>    endif()

Reply:
+1

Reply:
Thank you! Fixed in cd83762